.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "wasmCloud",
+    "image": "rust:bullseye",
+    "features": {
+        "devwasm.azurecr.io/dev-wasm/dev-wasm-feature/rust-wasi:0": {},
+        "ghcr.io/lee-orr/rusty-dev-containers/cargo-binstall:latest": {},
+        "ghcr.io/lee-orr/rusty-dev-containers/cargo-watch:latest": {},
+        "ghcr.io/lee-orr/rusty-dev-containers/cargo-nextest:latest": {},
+        "ghcr.io/devcontainers/features/nix": {}
```

Comment:
(original: https://github.com/wasmCloud/wasmCloud/pull/335#discussion_r1203946157)

> If we're adding this, does it support flakes out-of-the-box? Where can I find documentation around this?

Since dev containers are there for development purposes, I included nix since there is a flake.nix file in this repo, as well as tests running around nix in the CI, so I wanted to make sure that using files is supported within the dev container by default.
However - I have never used nix, so that is something you'd need to test separately.

Reply:
@rvolosatovs - can I consider this resolved?

Reply:
I think so, if it does not work out-of-the-box, then we (I) can address that in a follow-up!